broker-core/pom.xml

```diff
@@ -142,6 +153,24 @@
 			<artifactId>elasticsearch</artifactId>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.elasticsearch.client</groupId>
+			<artifactId>transport</artifactId>
+			<version>${elasticsearch-client-transport.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+	    	<groupId>org.apache.logging.log4j</groupId>
```

Comment:
So maybe just do it manually then. You can always enable "Show whitespaces" in Eclipse to see what's happening.